db/services.php

```diff
@@ -0,0 +1,181 @@
+        'type'        => 'read',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:viewhistory',
+    ],
+    'mod_kanban_get_discussion_update' => [
+        'classname'   => 'mod_kanban\external\get_kanban_content',
+        'methodname'  => 'get_discussion_update',
+        'description' => 'Retrieves the discussion for a card',
+        'type'        => 'read',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:view',
+    ],
+    'mod_kanban_change_kanban_content_add_column' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'add_column',
+        'description' => 'Adds a column to the kanban board',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:addcolumn',
+    ],
+    'mod_kanban_change_kanban_content_add_card' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'add_card',
+        'description' => 'Adds a card to a column of the kanban board',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:addcard',
+    ],
+    'mod_kanban_change_kanban_content_move_column' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'move_column',
+        'description' => 'Moves a column within the kanban board',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:managecolumns',
+    ],
+    'mod_kanban_change_kanban_content_move_card' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'move_card',
+        'description' => 'Moves a card within the kanban board',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:managecards',
+    ],
+    'mod_kanban_change_kanban_content_delete_column' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'delete_column',
+        'description' => 'Deletes a column and all contained cards from the kanban board',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:managecolumns',
+    ],
+    'mod_kanban_change_kanban_content_delete_card' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'delete_card',
+        'description' => 'Deletes a card from the kanban board',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:managecards',
+    ],
+    'mod_kanban_change_kanban_content_assign_user' => [
+        'classname'   => 'mod_kanban\external\change_kanban_content',
+        'methodname'  => 'assign_user',
+        'description' => 'Assigns a user to a card',
+        'type'        => 'write',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:assignself',
```

Comment:
is this the correct capability here? This external function is providing the assigning of all users, not just the current user himself, right?